client/commands.cc

```diff
@@ -261,13 +263,23 @@ command_create_post_xsnapshot(DBus::Connection& conn, const string& config_name,
 
 void
 command_delete_xsnapshots(DBus::Connection& conn, const string& config_name,
-			  list<unsigned int> nums)
+			  list<unsigned int> nums, bool verbose)
 {
     DBus::MessageMethodCall call(SERVICE, OBJECT, INTERFACE, "DeleteSnapshots");
 
     DBus::Hoho hoho(call);
     hoho << config_name << nums;
 
+    if (verbose) {
+	cout << _("Deleting snapshots from ") << config_name << ": ";
```

Comment:
Such sentence fragments can cause trouble for translators. It's better to use ```cout << sformat(_("Deleting snapshots from %s:"), config_name.c_str()) << endl;``` or even the plural form ```cout << sformat(_("Deleting snapshot from %s:", "Deleting snapshots from %s:", nums.size()),	config_name.c_str()) << endl;```.